colorbleed/plugins/maya/publish/validate_rendersettings.py

```diff
@@ -31,6 +37,11 @@ def process(self, instance):
         renderer = instance.data['renderer']
         layer_node = instance.data['setMembers']
 
+        # Main check animation
+        animation = cmds.getAttr("defaultRenderGlobals.animation")
```

Comment:
If you're checking per layer anyway - why does this need to also be checked globally? Just wondering.

Reply:
When we change renderlayer the access to the `defaultRenderGlobals` is still valid and the renderlayer applies any overrides to the `defaultRenderGlobals` if it has any. So if someone, for some reason decides to make an override to the `animation` to `Disabled` (which is possible) we catch it as well.

Reply:
Yes. I understand. But still, why then need to check it globally *before* switching to the renderlayer?

Reply:
After reconsidering my answer, you are correct. A bit weird as we also go over the `masterLayer`.